### ISSUE TITLE
Sleek theme's progress bar stays static when it is playing on a browser

### ISSUE THREAD

**Issue by Awesomecat713** [spicetify/spicetify-themes#506]
**Describe the bug**
On Sleek theme, the progress bar does not move up when you play the media on the Spotify web client.

**To Reproduce**
1. Open Spotify
2. Go to spotify.com and play music.
3. Go back to the Spotify client
4. The progress bar is misplaced.

**Expected behavior**
The progress bar is supposed to move upwards.

**Screenshots**
![progress bar](https://user-images.githubusercontent.com/64054522/130101274-65c7ae2c-26df-48b1-8ef2-46666ac5345f.png)

**Logs**
Add logs from console. To do that
1. Run `spicetify enable-devtools` in terminal
2. Spotify will be restarted
3. Hit <kbd>Ctrl + Shift + I</kbd> to open DevTools window
4. Navigate to tab Console
5. Copy console window content.

```console
Failed to load resource: the server responded with a status of 429 (Too Many Requests)
vendor~xpui.js:1 cosmos-connector:  fetchAdForSlot endpoint=%o sp://ads/v1/ads/leaderboard
vendor~xpui.js:1 MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 change listeners added. Use emitter.setMaxListeners() to increase limit
    at u (https://xpui.app.spotify.com/vendor~xpui.js:29:2781990)
    at t.o.addListener (https://xpui.app.spotify.com/vendor~xpui.js:29:2784272)
    at https://xpui.app.spotify.com/vendor~xpui.js:29:2090235
    at ka (https://xpui.app.spotify.com/vendor~xpui.js:29:3061153)
    at Ps (https://xpui.app.spotify.com/vendor~xpui.js:29:3085005)
    at t.unstable_runWithPriority (https://xpui.app.spotify.com/vendor~xpui.js:29:3296794)
    at hi (https://xpui.app.spotify.com/vendor~xpui.js:29:3030661)
    at Is (https://xpui.app.spotify.com/vendor~xpui.js:29:3084784)
    at ws (https://xpui.app.spotify.com/vendor~xpui.js:29:3069555)
    at vs (https://xpui.app.spotify.com/vendor~xpui.js:29:3068477)
    at https://xpui.app.spotify.com/vendor~xpui.js:29:3030933
    at t.unstable_runWithPriority (https://xpui.app.spotify.com/vendor~xpui.js:29:3296794)
    at hi (https://xpui.app.spotify.com/vendor~xpui.js:29:3030661)
    at yi (https://xpui.app.spotify.com/vendor~xpui.js:29:3030878)
    at vi (https://xpui.app.spotify.com/vendor~xpui.js:29:3030813)
    at hs (https://xpui.app.spotify.com/vendor~xpui.js:29:3067470)
    at Object.enqueueSetState (https://xpui.app.spotify.com/vendor~xpui.js:29:3035626)
    at r.A.setState (https://xpui.app.spotify.com/vendor~xpui.js:29:3220531)
    at r.<anonymous> (https://xpui.app.spotify.com/xpui.js:1:725072)
    at c (https://xpui.app.spotify.com/vendor~xpui.js:29:3265208)
    at Generator._invoke (https://xpui.app.spotify.com/vendor~xpui.js:29:3264996)
    at Generator.next (https://xpui.app.spotify.com/vendor~xpui.js:29:3265637)
    at r (https://xpui.app.spotify.com/vendor~xpui.js:29:3621511)
    at s (https://xpui.app.spotify.com/vendor~xpui.js:29:3621714)
(anonymous) @ vendor~xpui.js:1
pubads_impl_2021081801.js?31062329:6 [GPT] Invalid argument: PubAdsService.setRequestNonPersonalizedAds("0"). Valid values: 0,1.
xs.log @ pubads_impl_2021081801.js?31062329:6
pubads_impl_2021081801.js?31062329:6 [GPT] Invalid argument: PubadsService.setTagForUnderAgeOfConsent("0"). Valid values: 2,0,1.
xs.log @ pubads_impl_2021081801.js?31062329:6
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/service-worker.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/vendor~xpui.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/451.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui-desktop-modals.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
```

**Specifics (please complete the following information):**
 - OS: Windows
 - Spotify version: 1.1.66.578.gc54d0f69-a
 - Spicetify version: 2.5.0-itsmeow-patch4
 - Theme name: Sleek

**Comment by ajmeese7** [comment]
+1, I am experiencing the same problem

**Comment by CharlieS1103** [comment]
Just fixed this in Dreary theme 
<img width="1440" alt="Screen Shot 2021-08-20 at 1 17 52 AM" src="https://user-images.githubusercontent.com/60494758/130183129-1d5793aa-b869-4858-9965-29f9ede0f6e4.png">

**Comment by morpheusthewhite** [comment]
@harbassan

**Comment by harbassan** [comment]
Should be fixed now when the change is merged into the repo

**Comment by harbassan** [comment]
Try the most recent change, and make sure you've got the new files in the right location and have run spicetify apply. 

Also @Awesomecat713, are you still experiencing the issue aswell or is it fixed for you?

**Comment by harbassan** [comment]
As of rn, I think the issue might be coming from an issue within Spicetify itself so when the newest patch of that is out ill prbs be able to fix this aswell.

**Comment by ajmeese7** [comment]
I am still experiencing the issue after pulling the changes in 2585b6360b68452764fc25145f5cba1c75738fdf

**Comment by harbassan** [comment]
Yea, it has to do with the fact that the css mappings are unfinished as of right now, so only after that has been completed will the bug actually be fixed. 

You can try to temporarily fix it for yourself by replacing '.main-nowPlayingBar-container' in the css with the parent of the 'main-connectBar-connectBar' in the following line of code:
```
.now-playing-bar-container>div:first-child:nth-last-child(2) .playback-bar, .main-nowPlayingBar-container>div:first-child:nth-last-child(2) .playback-bar {
  bottom: 107px;
}
```

**Comment by afonsojramos** [comment]
Closing this as the problem was related to CSS Mappings.
